### PR TITLE
Add xDai STAKE strategy

### PR DESCRIPTION
**Pull request by maxaleks**
Changes proposed in this pull request:
- added a strategy that calculates the user's voting power based on the balance of STAKE tokens and the amount of tokens deposited on EasyStaking together with the accrued emission

### CODE REVIEW

File: src/strategies/xdai-stake/index.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    easyStakingDeposits,
+    { sigmoidParameters, totalSupplyFactor, totalStaked },
+    erc20Score,
+    block,
+    totalSupply,
+  ] = await Promise.all([
+    getEasyStakingDeposits(network, addresses, snapshot),
+    getEasyStakingParams(network, snapshot),
+    erc20BalanceOfStrategy(space, network, provider, addresses, options, snapshot),
+    getBlock(provider, snapshot),
```
Comment (bonustrack) [inline src/strategies/xdai-stake/index.ts:115]: You can use `provider.getBlock(snapshot)` directly the new function `getBlock` is not necessary.

**Review by bonustrack — COMMENTED** [review]
Looks good! Can you just do the suggestion and run the script "lint" with `npm run lin`? Thanks!

File: src/strategies/xdai-stake/index.ts (hunk 1 above)
Comment (maxaleks) [reply src/strategies/xdai-stake/index.ts]: You're right, sorry 😅

**Comment by maxaleks** [comment]
> Looks good! Can you just do the suggestion and run the script "lint" with `npm run lin`? Thanks!

Fixed!

**Review by bonustrack — APPROVED** [review]

**Pull request merged**